M/MEOS/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MEOS"
+version = v"1.2.0"
+
+# MEOS (Mobility Engine, Open Source) is a standalone C library and the core component of MobilityDB.
+# For further information, see https://libmeos.org.
+sources = [
+    GitSource("https://github.com/MobilityDB/MobilityDB.git", "60048b5b4b7ce2f7560c024d1af024db73b3bd5b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/MobilityDB
+
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DMEOS=ON \
+    -DHAVE_X86_64_POPCNTQ_EXITCODE="FAILED_TO_RUN" \
+    -DHAVE_X86_64_POPCNTQ_EXITCODE__TRYRUN_OUTPUT=
```

Comment:
Why?  This is such an old instruction which should be moderately safe to assume to be available.

Reply:
Apparently it's not the instruction, but the TRY_RUN statement is not allowed in cross-compile mode:
```
[14:57:59] CMake Error: TRY_RUN() invoked in cross-compiling mode, please set the following cache variables appropriately:
[14:57:59]    HAVE_X86_64_POPCNTQ_EXITCODE (advanced)
[14:57:59]    HAVE_X86_64_POPCNTQ_EXITCODE__TRYRUN_OUTPUT (advanced)
[14:57:59] For details see /workspace/srcdir/MobilityDB/build/TryRunResults.cmake
```

Reply:
> but the TRY_RUN statement is not allowed in cross-compile mode:

I know.  But I expect that try-run is running a check whether the instruction is available.  Which of course could also be done with a simple compiler check instead of a try-run.

Reply:
I just workarounded the error with the instruction CMake gave me (I'm not affiliated with MobilityDB/MEOS so I can't answer the why, but I'm happy to make an upstream issue). AFAIK there's no code in MEOS actually making use of this check (anymore https://github.com/MobilityDB/MobilityDB/pull/610).

Reply:
We use a cross-compilation environment, so any try_run call in a CMake script will basically be pointless. From the CMake docs (https://cmake.org/cmake/help/latest/command/try_run.html#behavior-when-cross-compiling):

> When cross compiling, the executable compiled in the first step usually cannot be run on the build host. The try_run command checks the [CMAKE_CROSSCOMPILING](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html#variable:CMAKE_CROSSCOMPILING) variable to detect whether CMake is in cross-compiling mode. If that is the case, it will still try to compile the executable, but it will not try to run the executable unless the [CMAKE_CROSSCOMPILING_EMULATOR](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING_EMULATOR.html#variable:CMAKE_CROSSCOMPILING_EMULATOR) variable is set. Instead it will create cache variables which must be filled by the user or by presetting them in some CMake script file to the values the executable would have produced if it had been run on its actual target platform. These cache entries are: